numpydoc/validate.py

```diff
@@ -633,7 +633,23 @@ def validate(obj_name, validator_cls=None, **validator_kwargs):
 
     errs = []
     if not doc.raw_doc:
-        if "GL08" not in ignore_validation_comments:
+        report_GL08: bool = True
+        # Check if the object is a class and has a docstring in the constructor
+        if doc.name.endswith("__init__") and doc.is_function_or_method:
+            cls_name = doc.code_obj.__qualname__.split(".")[0]
+            cls = getattr(importlib.import_module(doc.code_obj.__module__), cls_name)
```

Comment:
@larsoner I don't know enough about current internals to know if this is how we do things; no utility functions / class methods for this purpose?

Reply:
Looks like `Validator._load_obj(f"{doc.code_obj.__module__}.{doc.code_obj.__qualname__}")`  might do it, or maybe simpler `Validator._load_obj(doc.name[:-9])` could work. @mattgebert do you want to try it (probably the second one)?